controllers/index.js

```diff
@@ -11,13 +11,18 @@ exports.getCategories = (req, res) => {
     });
 };
 
-exports.getReviewById = (req, res) => {
+exports.getReviewById = (req, res, next) => {
   const { review_id } = req.params;
   selectReviewById(review_id)
     .then((review) => {
-      res.status(200).send({ review });
+      if (review === undefined) {
+        return Promise.reject({
+          status: 404,
+          msg: `No review found for review_id: ${review_id}`,
```

Comment:
Be careful with really precise error messages as this can give away info about your db